ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dot1x.j2

```diff
@@ -0,0 +1,24 @@
+{# DOT1X Interfaces #}
+{% set ethernet_interfaces_dot1x = [] %}
+{% for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
```

Comment:
Since this PR was prepared, we have modified the ethernet_interfaces templates so be compatible with future datamodel change. Check out PR #1639, on how this was changed for some of the other features. `qos-interfaces.j2` should have similar structure to this, so you can get some "inspiration" :)

Reply:
completed per the example from qos-interfaces.j2